### DISCUSSION TITLE
🌌 Hacking Under The Stars (Summer, 2023)

### DISCUSSION THREAD

**Discussion by bellesea** [hackclub/hackclub#1844] [General] [answered]
### 👋 Hey there!

We're so glad you're here. Comment below to RSVP for [Hack Club](https://hackclub.com)'s 2023 summer event (check out last year's and the year before that!) and be the first to hear when signups go live.

We'd like to know, **what's one thing you’d want to do if you had seventy-two hours in the woods?**

The entire adventure will be community-built and open-sourced - upvote and comment on your favourite ideas so we can build a hackers' oasis together.

_(Don't just respond with just RSVP! We'd love to hear your ideas.)_

<img src="https://user-images.githubusercontent.com/39828164/236823165-1b5c32e8-4143-4b36-b5ff-a76be2f1ac65.jpeg" width="600px" />

**Comment by alexdevz** [comment]
I would enjoy a mixture of hacking out some projects I've been working on and connecting with new friends with nature all around us :)

**Reply by YashveerSingh1062** [reply]
RSVP, excited to learn new things between geniuses..

**Reply by fenestbuc** [reply]
RSVP, sounds fun

**Reply by BinaryAP** [reply]
RSVP, would love to improve my knowledge and cement my love for coding. I'm into exploring and applying good use of my time, so I'd do something tangible and creative with my time in the woods

**Comment by sampoder** [comment]
I've been really enjoying ultimate frisbee lately; I'd love to play some over the weekend! At Assemble, I really enjoyed talking to Hack Clubbers and getting to know them personally. I'd love to have more interesting conversations whilst in the woods.

**Reply by zeyuyaoy** [reply]
Frisbee!!

**Reply by fenestbuc** [reply]
+1

**Reply by CheeksTheGeek** [reply]
yea yay yea im up for frisbee too

**Reply by prince-chhirolya** [reply]
I want to join this.

**Comment by hrithikthakur** [comment]
learning survival skills such as fire-building and shelter-making and building technology to aid in sending distress signals :)

**Comment by anishsaraogi** [comment]
Coding while looking at the stars >>>>

**Comment by tddschn** [comment]
RSVP

**Reply by eunjihu** [reply]
RSVP

**Comment by tddschn** [comment]
RSVP

**Comment by cfanoulis** [comment]
cook marshmallows, maybe make a bot to cook em

**Comment by zhuofan-16** [comment]
RSVP

**Comment by HenryBass** [comment]
RSVP!!!

**Comment by Linus-Lu** [comment]
Hiking, climbing and maybe some coding under the stars

**Comment by Linus-Lu** [comment]
Hiking, climbing and maybe some coding under the stars

**Comment by nobiit** [comment]
RSVP

**Comment by frankonyemaorji** [comment]
I want to enjoy the beautiful nature of forest wildlife. RSVP

**Comment by debater-coder** [comment]
I would make some physical user interfaces that suit the environment!

**Comment by frankonyemaorji** [comment]
I want to enjoy the beautiful nature of forest wildlife. RSVP

**Reply by anamiikajha** [reply]
RSVP, want to have fun while learning

**Comment by polypixeldev** [comment]
I would like to enjoy the nature and have fun coding with other hackers!
_also launching a satellite sounds fun_

**Reply by sampoder** [reply]
that's ambitious!

**Reply by faisalsayed10** [reply]
moonbeam 👀

**Comment by louremipsum** [comment]
you mean to say that I can Touch grass AAND talk with like-minded people/ build something
RSVP me

**Reply by sampoder** [reply]
yeah!!

**Comment by alexanderhm95** [comment]
Rsvp

**Comment by bharath-kumar143** [comment]
To just get surround with the best contributor with the whole around the Woods and the fragnance of it!!

**Comment by HEYKASHIF** [comment]
I am new to this but excited too 😁

**Comment by akashnigam-0** [comment]
RSVP

**Comment by akashnigam-0** [comment]
RSVP

**Comment by poonam-16** [comment]
Network and enjoy the nature, a lot of new ideas strikes up in our mind just by looking at nature and then try and build the solution for the same. Who knows we might be able to solve one of the major problems during this period. :)

**Comment by festusmaithyakcau** [comment]
I would engage in learning survival tricks and also self actualization by bonding with nature 🤗🤗🤗

**Reply by festusmaithyakcau** [reply]
RSVP

**Comment by Ahmed-aleryani** [comment]
Count me in
RSVP

**Comment by dem0nsl4yer** [comment]
RSVP done.

**Comment by VS11000** [comment]
excited to be part of it
RSVP

**Comment by satishmane-sam** [comment]
> ### 👋 Hey there!
> We're so glad you're here. Comment below to RSVP for [Hack Club](https://hackclub.com)'s 2023 summer event (check out last year's and the year before that!) and be the first to hear when signups go live.
> 
> We'd like to know, **what's one thing you’d want to do if you had seventy-two hours in the woods?**
> 
> The entire adventure will be community-built and open-sourced - upvote and comment on your favourite ideas so we can build a hackers' oasis together.
> 
> <img alt="" width="600px" src="https://user-images.githubusercontent.com/39828164/236823165-1b5c32e8-4143-4b36-b5ff-a76be2f1ac65.jpeg">

exited to be the part of this community

**Comment by reesericci** [comment]
To answer the question, I would be trying to get out of the woods asap lol

**Reply by sampoder** [reply]
I'll race you - first to JFK on foot from the woods gets to be the "Hack King"? /s

**Comment by reesericci** [comment]
RSVP

**Comment by JosiasAurel** [comment]
I really love looking at trees and breathing fresh air. I'd love to connect with other HackClubbers in this *different* space and talk about things unrelated to coding though I'd love to also hack some cool stuff with them in the heights.

**Comment by Younus-Saberi** [comment]
RSVP

**Comment by akashnigam-0** [comment]
Sure we'll do it

**Comment by kkg2001** [comment]
RSVP

**Comment by Apoorvgarg-creator** [comment]
I would like to enjoy the nature and have fun coding with fellow coders !

**Comment by sobhanbera** [comment]
I would love to join you guys!

**Comment by SANTHOSH-MAMIDISETTI** [comment]
**RSVP**

**Comment by hariketsheth** [comment]
RSVP

**Comment by Sid-WC121** [comment]
Excited to join you guys
RSVP

**Comment by Justine-Njuguna** [comment]
Code until I drop. RSVP.

**Comment by InternetRamen** [comment]
Hi Belle! Prob can't go this year but looking forward to hearing more

**Reply by bellesea** [reply]
Maybe next year! :(

**Comment by incchi** [comment]
hey over here :)

**Comment by Arpan-206** [comment]
I would love to work on my astronomy related 3D project under the stars.

**Reply by sampoder** [reply]
ooo! what's your project about?

**Comment by ghost** [comment]
> ### 👋 Hey there!
> We're so glad you're here. Comment below to RSVP for [Hack Club](https://hackclub.com)'s 2023 summer event (check out last year's and the year before that!) and be the first to hear when signups go live.
> 
> We'd like to know, **what's one thing you’d want to do if you had seventy-two hours in the woods?**
> 
> The entire adventure will be community-built and open-sourced - upvote and comment on your favourite ideas so we can build a hackers' oasis together.
> 
> _Please do not respond with just RVSP_
> 
> <img alt="" width="600px" src="https://user-images.githubusercontent.com/39828164/236823165-1b5c32e8-4143-4b36-b5ff-a76be2f1ac65.jpeg">

I would be so happy to be part of an outdoor hack ,I want to RSVP for this event , and Totally looking forward for this event

**Comment by moultamissimane** [comment]
Hi, I want to RSVP for this event!

**Comment by el-tegy** [comment]
Hello, RSVP !!

**Comment by ItsRoy69** [comment]
I would love to play in the woods,  engage with nature and the residents of that particular place, enjoy the serenity.
RSVP

**Comment by BolunThompson** [comment]
Coding, hiking, reading, cooking, playing!

**Comment by Ritesh-zt0** [comment]
Nature time and meeting with new people and understanding new things. Sounds exciting, RSVP

**Comment by PasinduDushan** [comment]
I would become friends with everyone in the meet as well as code weird web apps together. Also, I would relax while looking at the beautiful night sky

**Comment by minhadona** [comment]
build a simple night camera auto rechargeable with the sun (it has to stay in the leaves high up the trees) to film the fauna (mostly birds)!!!! 

but i'm in brazil, no money to go. if someone is interested, i'll be online to contribute!!!

**Comment by MitAbhay** [comment]
Group hike or trail run, yoga or meditation, code jam, scavenger hunt and many more excited things :) RSVP

**Reply by bellesea** [reply]
omg scavenger hunts!!!

**Comment by Kushgupta16** [comment]
Looking forward for this event ✌️

**Comment by harsha-sam** [comment]
Hey ! Looking forward for the event. Code until I drop !

**Comment by Nakshatraarora** [comment]
I would like to breathe some fresh air and maybe let my creative mind churn awesome ideas to work on for the week.
RSVP

**Comment by Redskull-127** [comment]
Imagining RSVP to hackathon with GitHub mascot under the stars >>>> ✨

**Reply by bellesea** [reply]
yesss — we need a mascot at the hackathon!!

**Comment by Abhaykushwah** [comment]
> ### 👋 Hey there!
> 
> We're so glad you're here. Comment below to RSVP for [Hack Club](https://hackclub.com)'s 2023 summer event (check out last year's and the year before that!) and be the first to hear when signups go live.
> 
> We'd like to know, **what's one thing you’d want to do if you had seventy-two hours in the woods?**
> 
> The entire adventure will be community-built and open-sourced - upvote and comment on your favourite ideas so we can build a hackers' oasis together.
> 
> _(Don't just respond with just RVSP! We'd love to hear your ideas.)_
> 
> <img src="https://user-images.githubusercontent.com/39828164/236823165-1b5c32e8-4143-4b36-b5ff-a76be2f1ac65.jpeg" width="600px" />
> 
> 


RSVP for Hack Club's 2023 summer event

**Reply by amankumar003** [reply]
Really a great event , want to be part of such program 
RSVP

**Comment by bawneet04** [comment]
If I had 72 hours in woods, then I would be doing "Hiking". Exploring the trails and pathways in the woods, appreciating the natural beauty, and discovering different flora and fauna along the way.

**Comment by alx-alexpark** [comment]
RSVP (rust file extension vice president)

I think having a zephyrnet style server or network would be very cool.

**Comment by Devnol** [comment]
How about a drone attachment that uses thermal imaging and opencv to identify top-downs of wild animals that we then befriend <3

**Comment by MYaserA** [comment]
RSVP :)

**Comment by SamuelATAKE** [comment]
I would really be pleased being part of such a huge opportunity to make to life awesome projects / plateforms on education in order to help so many persons. Why not all this with some AI stuffs...
RSVP.

**Comment by sanjay-kv** [comment]
Camping and coding under stars.

**Comment by jayeshrajpoot** [comment]
if I had seventy-two hours in the woods I would definitely try to build tech that will have a inclusive impact on all human beings and make their life "a little bit" better.

**Comment by DEABSTRACT** [comment]
RSVP ):

**Comment by briannamagtoto** [comment]
stargazing! and hiking ofc

**Comment by Ashwagandhae** [comment]
stargaxing & hiking

**Comment by primitt** [comment]
I would like to come to this event, though I have no explicit ideas atm, I do see a lot of good ones in the comments above! This looks like it will be a great event! (maybe there can even be "sports" competitions in the middle of the hackathon lol)

**Comment by chanderbhanu096** [comment]
RSVP, hiking

**Comment by jyotikaisrani** [comment]
RSVP, STAR GAZING!!

**Comment by Nick67644** [comment]
I say we should go for a hacking hike! :D

**Comment by phanturne** [comment]
I'd love to participate! :)

**Comment by Z3r0J** [comment]
RSVP

**Comment by w0110** [comment]
RSVP
Coding under a sky full of stars!

**Comment by MHKhan18** [comment]
RSVP

**Comment by TirthPatel06** [comment]
RSVP! If i get 72 hours in wood which will be very Extraordinary and refreshing for me, i will love to talk with my friends there and do meditation. The peace and contentment of the place like that will always attract <3

**Comment by DuanBoomer** [comment]
RSVP ( Rust Sql Vue Python )
That's a tech stack I would explore in this event.
Everyone be ready 🤫

**Comment by Darkridersia** [comment]
> ### 👋 Hey there!
> We're so glad you're here. Comment below to RSVP for [Hack Club](https://hackclub.com)'s 2023 summer event (check out last year's and the year before that!) and be the first to hear when signups go live.
> 
> We'd like to know, **what's one thing you’d want to do if you had seventy-two hours in the woods?**
> 
> The entire adventure will be community-built and open-sourced - upvote and comment on your favourite ideas so we can build a hackers' oasis together.
> 
> _(Don't just respond with just RSVP! We'd love to hear your ideas.)_
> 
> <img alt="" width="600px" src="https://user-images.githubusercontent.com/39828164/236823165-1b5c32e8-4143-4b36-b5ff-a76be2f1ac65.jpeg">

I would like to learn new skills and expand my knowledge on coding ;)

**Comment by rimwaka** [comment]
1. RSVP.
2. Hiking trails with the local community, learn their ways of life and appreciate their culture. Maybe we might code and solve one of the issues they are facing.
3. Bonfire and stars counting.
4. Thank you.

**Comment by adarssh01** [comment]
RSVP! 
I would love to contribute into open source and I want to connect with new people, share ideas with them and also learn a lot ....
I want to stare at the moon and stars all night at the woods 🫠

**Comment by Jony-Jas** [comment]
Hear horror stories from th colleagues 👻

RSVP.

**Comment by Nikhil8557** [comment]
#1844 #1846 
### **_We can make cook up some stories, Lassi and Pratha in breakfast. And some hide and seek games 😶‍🌫️_**
RSVP

**Comment by ANSXX** [comment]
Learning survival skills, cooking in the wilderness, and making new friends sounds amazing.

**Comment by ManasG0yal** [comment]
RSVP

As I was growing I always want to connect with people and new groups but due to my shyness and not being able to connect as i grow, I came to know the value of such group activities.

As a fellow coder from India, I want to be part of this happening activities group,
-> I am good at cooking and packing everyone with the spicy cuisine of India.🌶️🫚
-> We all can play cricket in the streets ( gully cricket ) style which is a wholesome activity in itself.🏏
-> Learn from everyone and connect with all.🛜🎮
-> We sit down in the open field and code and play games.
-> also to learn some skills which help me to fight crocss 😂😂

**Comment by AMSANJEEV28** [comment]
Adventureeeeeeeeee 🚀 RSVP

**Comment by VipulChauhan89** [comment]
Bonfire 🔥🔥

RSVP.

**Comment by CheeksTheGeek** [comment]
72 hours equals fraternizing with devs + developing an indie 4v4 game with raffle-based team matching + badminton + maybe making a glow-at-night shuttlecock using some phosphorescent powder + making memories for life 🤩🤩🤩

also, obviously I'd like to RSVP too since I see people around doing that

**Comment by AbishekRufusRaj** [comment]
RSVP! Would love to learn new skills and expand my knowledge by the bonfire! Spend time connecting with people under the trees 🌲

**Comment by ashutosh7i** [comment]
RSVP, excited to make my own GPS enabled security device.🥳

**Comment by thereavercode** [comment]
RSVP : I've always likened coding to poetry, and the image that comes to mind just thinking about how congruent this event will prove this to be makes me eager to experience it.

**Comment by Shaikh-Yaqoob** [comment]
RSVP! Would love to learn new skills and expand my knowledge by the bonfire!

**Comment by dnyaneshwar-rede** [comment]
RVSP 

READY FOR NEXT ADVENTURE  💥

**Comment by hwittenborn** [comment]
RSVP! Just doing some normal camping stuff would be pretty amazing - there aren't a ton of active programmers at my high school, and a chance to meet with fellow programmers would just be a good time in general!

**Reply by sampoder** [reply]
That's awesome to hear, nice to meet you @hwittenborn!

**Comment by harsh721827** [comment]
super excited for this

**Comment by Parthiv-01** [comment]
RSVP

**Comment by julianalg** [comment]
RSVP! making smores would be fun :D

**Comment by phillipe-clinton** [comment]
Would like to explore and meet new folks to work together. 
currently working on a proof of algorithm consensus mechanism for VANETs! 
need contributors and sponsors!

**Comment by shiroyasha9** [comment]
RSVP! Would love to collab on some astronomy related application gazing at the stars while having a hot cup of coffee and barbeque 😋

**Comment by allensphilip** [comment]
RSVP! I would like to be in it

**Comment by Athul0491** [comment]
RSVP! Would be nice to explore and meet new people, over coffee and bonfire, building some cool stuff while stargazing🥳

**Comment by DevIos01** [comment]
RSVP! 

some kind of Scavenger hunt, where you leave some low-frequency Bluetooth connection and we have to find it through gadget we have, would be interested maybe in some kind of ctf outdoor

**Reply by sampoder** [reply]
i like that idea! let's make it happen!

**Reply by karmanyaahm** [reply]
LoRa?

**Comment by Rhyzhang** [comment]
RSVP

Code and discuss the future of humanity under the stars and next to the campfire!

**Comment by Shield01** [comment]
Enjoy the wildlife and make friends while building a product that would help people donate to wildlife preservation.

**Comment by ebilesh** [comment]
I would climb a tree (preferably successfully) and make new friends while trying something new
RSVP

**Comment by SerenityUX** [comment]
RSVP, let's do a hike to the camp site :)

**Comment by recursiveforte** [comment]
RSVP!

**Comment by alexbluo** [comment]
rsvp - fishing fish, farming food, fighting bears

**Comment by karmanyaahm** [comment]
Break out the telescope I haven't used in years. I wonder what connectivity we'll have :thinking: . I'd love to do push notifications over satellite :satellite: but that seems too expensive.

**Comment by hyusap** [comment]
I would love to go biking throughout the forest! and ofc meeting with the community and working on (completely offline?) projects, so RSVP!

**Comment by Leo32345** [comment]
This sounds awesome! Personally I would:
1. Do a project that is related to the forest/environment 🏕️ 
2. Go hiking, anywhere near streams/mountains sounds great
3. Take a bunch of photos and try to make new friends
4. Probably come up with something to do on the fly

Also, because everyone else is saying it… RSVP!

**Comment by ganning127** [comment]
Hi, I'm Ganning! I'm a high school student in North Carolina and lead a hack club at my school. I would love to build a project while meeting people who also love "hacking" together fun things

**Comment by NisargWath** [comment]
Like to join some AI stuff with real world problems , RSVP

**Comment by JerrySantana** [comment]
RSVP! Laugh, sing, learn and of course CODE! ❤️🙌🏽

**Comment by archi-max** [comment]
Augmented reality and nature sound like something I would love to dive into. RSVP!

**Comment by Crade-IN** [comment]
RSVP! ❤️

**Comment by ZuhairHossain** [comment]
RSVP!
Want to spend time camping, hiking, and cooking meals over an open fire as well develop new things together as a team.

**Comment by NalinPlad** [comment]
RSVP 🤩
Going to Assemble in my home town was the most fun I had ever had programming, and the first event I even went to where I connected with people who loved programming as much as I do. Assemble was my first hackathon and i've gone to several more after it, and going to this one would definitely be the highlight of my summer! Camping and being outdoors is also one of my favorite activities, I co-own my schools trail club so camping and hiking and in general being outdoors is a lot of fun

One thing I'd love to do out in the woods would be to try and build or implement a long distance communication protocol or system using some raspberry pis and radio recievers I have laying around, for communication.

**Reply by sampoder** [reply]
I'm so glad to hear @NalinPlad! Your radio idea sounds sick.

**Comment by mittal-parth** [comment]
Bonfire + a game of treasure hunt maybe, along with building the coolest products! 
RSVP!

**Comment by YusufHass** [comment]
RSVP! Excited to expand my knowledge

**Comment by avanoxious** [comment]
I'd like to make new friends and hang out with people in the community.
(RSVP)

**Comment by ArunodaySingh** [comment]
I'd love to join you guys (RSVP)

**Comment by ritikaradh** [comment]
I would look up to the stars in the clear sky... Feel the woods and its creatures... Learn from their ways  of living and behaving... And would try to make something that would show human's kindness towards the nature. 
Rsvp!

**Comment by atul120212** [comment]
RSVP excited to join

**Comment by Darshan4141** [comment]
RVSP! I am really excited to code under the stars!!

**Comment by Otherwa** [comment]
RSVP! fuck it we ball

**Comment by nikitam25** [comment]
RSVP!!! (hacking :))

**Comment by Savalone47** [comment]
Thanks to all :)

**Comment by AkulxSharma** [comment]
RSVP! would love to learn new things from the people I meet

**Comment by sanjus-robotic-studio** [comment]
RSVP

**Reply by sanjus-robotic-studio** [reply]
It would be awesome to build a wakie takie or a radio transmitter and receiver. For communication during trecking

**Comment by briyandyju09** [comment]
Spend time with the best people in the world under the stars while coding a robot or something 💖

**Comment by ongzzzzzz** [comment]
RSVP!! I'd love to make awesome projects with awesome people under the starry night - it would combine my passion in making with my interests in stargazing :D

**Reply by bellesea** [reply]
stargazing yes!!!

**Comment by ryanchou-dev** [comment]
rsvp o.O

would love to hang out and eat food w y'all!

**Comment by NilanchalaPanda** [comment]
I would enjoy nature's beauty and also do some productive tasks; would be great connecting with like-minded people as well :)

**Comment by rajansagarwal** [comment]
offline hacks! learning how to build and create products without connection to an internet (ie Swift's MultiPeer Connectivity, etc...) 👀 (also rsvp!)

**Reply by bellesea** [reply]
omg yes! our own network!!!

**Comment by ZeroQLi** [comment]
SIGN ME UP. I am definitely not an outdoor person but being around like minded people will definitely help me open up.

As for the woods, Anyone got a telescope?

**Comment by nesrina-lgr** [comment]
I would love the idea of coding while meeting like-minded people in such a beautiful place!

**Comment by ANURAG-KUMAR-4** [comment]
RSVP- Lying on ground and seeing stars at night time/ having fun near bonfire, exploring forest and finding new things and facts about the forest.

**Comment by samarth641** [comment]
RSVP Coding And Stars hell ya !!!!

**Comment by jasAtbiz** [comment]
Hey can anyone tell me how to register for this 
Is travel and accomodation provided for folks who are travelling from different countries ?

**Reply by cfanoulis** [reply]
Hey! There will be more news on 12/05. Join the Slack to keep in the know :) https://hackclub.com/slack

**Comment by VipulChauhan89** [comment]
RSVP! Would love to learn new skills and expand my knowledge like the bonfire.Sent from my iPadOn 09-May-2023, at 11:59 PM, Mohammad Yaqoob ***@***.***> wrote:﻿
RSVP! Would love to learn new skills and expand my knowledge by the bonfire!

**Comment by tiluckdave** [comment]
RSVP. But please ensure you unlimited supply of coffee first 🚀

**Comment by Adityabhaskar685** [comment]
RSVP

**Comment by val-hir** [comment]
Let’s get some telescopes out there!

**Reply by LordUbuntu** [reply]
What if we made the telescopes follow the motion of a specific celestial body?

**Comment by LordUbuntu** [comment]
I think it would be really cool to hike a mountain and then contact the people on the ground with a radio we built earlier.

**Comment by PeterTheSalmon** [comment]
I'd love to go out canoeing or kayaking!

**Comment by ClaireBookworm** [comment]
You should have a survival challenge! Maybe try to have people try to navigate their way back to base from a place far away? Or get people to climb a bunch of trees :). There's also gotta be guitar lessons and campfire songs!!

**Reply by bellesea** [reply]
haha yes, try not to die 😳

**Comment by Peiprjs** [comment]
I would really enjoy doing some sports out in the open, as a way to have a break from coding. Also, it'd be really cool to have a few telescopes... (RSVPing btw)

**Comment by NikSchaefer** [comment]
RSVP

Let's code under the stars ⭐

**Reply by bellesea** [reply]
hack yeah!

**Comment by Yaduttam95** [comment]
> ### 👋 Hey there!
> We're so glad you're here. Comment below to RSVP for [Hack Club](https://hackclub.com)'s 2023 summer event (check out last year's and the year before that!) and be the first to hear when signups go live.
> 
> We'd like to know, **what's one thing you’d want to do if you had seventy-two hours in the woods?**
> 
> The entire adventure will be community-built and open-sourced - upvote and comment on your favourite ideas so we can build a hackers' oasis together.
> 
> _(Don't just respond with just RSVP! We'd love to hear your ideas.)_
> 
> <img alt="" width="600px" src="https://user-images.githubusercontent.com/39828164/236823165-1b5c32e8-4143-4b36-b5ff-a76be2f1ac65.jpeg">

RSVP

Excited to meet new and amzing minds... ✨✨

**Comment by mudit13** [comment]
A hidden quest for people divided in groups based on their interests , volleyball and other sports. RSVP

**Reply by bellesea** [reply]
ooooo hidden quests sound fun!

**Comment by singhayush4331** [comment]
Hey there 👋
It's too much fun when you got to interact with so many people with very different mindset or the take on things and you grow as a person! 
So that's why I am excited!! RSVP

**Comment by priyankasaini69** [comment]
HOLA AmiGOS 👋
I'm too excited to get to interact with so many people with different  mindset
I am excited to build something like chatgpt and even more better than that
which can resolve the problems of people easily 
haha 
I'm just too excited
and working with these many people from around the world 
this is even more exciting
RSVP!!

**Reply by bellesea** [reply]
yes, we can't wait to build with you :)

**Comment by alx-alexpark** [comment]
Ham radio would be very cool.

**Reply by bellesea** [reply]
it would!!!

**Comment by priyankasaini69** [comment]
Hell!!yeah

**Comment by Aliyugambo** [comment]
RSVP!. I would enjoy hacking and also exploring new places, meeting new people and connecting together by shearing and bring new ideas to live.

**Comment by yokwejuste** [comment]
Wow such an amazing opportunity, I'm Steve, 

Sendings my greetings right from Cameroon. Will be happy joining the sessions and coming over.

**Comment by riju951** [comment]
RVSP

**Comment by pengziyi-handsome** [comment]
On the premise of ensuring safety, I think I will explore the world with curiosity like a child, because I have rarely or for a long time not had close contact with nature, and I miss her very much.

**Comment by Swarthe** [comment]
RSVP

hello!

I would love to play badminton or table tennis in the woods (or any other racquet sport). It would also be great to see what projects others are working on.

**Comment by Seun-Ajayi** [comment]
RSVP

I would love to learn new technologies and explore existing ones while connecting with industries expert, professionals and peers.

**Comment by OmarUsman** [comment]
RVSP!! Survival challenge sounds perfect.

**Comment by Daksh777** [comment]
RSVP!

Would love to sing songs, play frisbee, talk with people and code under the stars

**Comment by jasmina-dev** [comment]
do some cooking over a campfire!! +RSVP

**Comment by NishkalHundia** [comment]
RSVP

**Comment by SameerYasin** [comment]
RSVP!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by anony-mouse1** [comment]
RSVP!

It'd be great to have a hike - and -code adventure, where we switch between hiking and coding to stay mentally and physically productive

**Comment by aidenybai** [comment]
hear me out: orpheus cult meeting

**Comment by melindachang** [comment]
RSVP. hack club hike

**Comment by ako-001** [comment]
RSVP! This sounds like a lot of fun, and I'd like to make some friends.

**Comment by ArunodaySingh** [comment]
Can anyone please let me know what going to have happened why we join here?

**Comment by Srivaths-P** [comment]
I'd think about life and philosophy. Maybe make some friends and play some physical games too for once in my life. RSVP

**Comment by LinusSkucas** [comment]
Tree climbing is cool; why not combining that with an altimeter?

**Comment by jaiswalrobin** [comment]
RSVP, I would love to build and explore with new people

**Comment by SameerYasin** [comment]
RSVP!! This is perfect! It will be amazing to go hiking, and maybe play
some sport! and even coding stuff out in the open! I've seen pictures of
Vermont and I'm so down to go around there!!

**Comment by captainexpo** [comment]
RSVP, this will be great!

**Comment by fenestbuc** [comment]
RSVP, this will be fun. We can have stargazing activities as well !!

**Comment by lucaspotter** [comment]
RSVP :)) something with amateur radio would be great!

**Comment by Noahk2024** [comment]
RSVP! I think that a LAN gaming tournament or something similar would be really fun!

**Comment by se1yu** [comment]
RSVP :); I'd love campfires

**Comment by mw-xyz** [comment]
RSVP, I'd love late-night campfire stories with s'mores - ik this is so cliche but it's still a vibe!

**Comment by alexdevz** [comment, marked as answer]
Hey there RSVPs! 

Today’s the day; **sign ups have opened for this summer’s event at [outernet.hackclub.com](https://outernet.hackclub.com/)**.

If you hadn’t figured it out yet, this summer, we’re going camping in the Northeast Kingdom of Vermont. We hope that you can join us and make this the most meaningful weekend of the year.

**On July 28th, we’ll gather in Cabot, Vermont to begin our grand experiment - creating a hacker oasis.** For four days and three nights, we’ll turn a humble campground; Covenant Hills Campground, into an awesome space for hackers of all kinds.

![IMG_9601](https://github.com/hackclub/hackclub/assets/38901010/09c6e933-b1db-4ae2-ba72-7bc10a6886a9)

**You choose your own adventure.** Want to make stupid shit nobody needs? There’ll be a hackathon of sorts. Want to learn Rust? Check out #outernet-lab. Want to chat about why Error 418 is critical to the modern internet? Join #outernet-hike-club or grab a drink at the #outernet-cafe. You name it, you’ll have the space to do it.

At the end of the day, **your experience at Outernet will be defined by the things you build, the adventures you go on, and the friends you make.** Through guilds, we’ll come together to create the perfect environment for one another to make things. Contribute to a guild in #outernet-guilds or start your own at [github.com/hackclub/outernet](https://github.com/hackclub/outernet). And don’t forget, us at HQ always have a trick or two up our sleeves.

**Head over to [outernet.hackclub.com](https://outernet.hackclub.com/) to register and learn more;** as you sign up, think about how you can make the most of your time at Outernet. Let’s come together to make this trip incredible.

cc: @auroralia @se1yu @Noahk2024 @lucaspotter @fenestbuc @Captainexpo-1 @SameerYasin @jaiswalrobin @LinusSkucas @CrapTheCoder @ArunodaySingh @ako-001 @melindachang @aidenybai @anony-mouse1 @SameerYasin @NishkalHundia @mooghoo @Daksh777 @OmarUsman @Seun-Ajayi @Swarthe @pengziyi-handsome @riju951 @yokwejuste @Aliyugambo @priyankasaini69 @alx-alexpark @singhayush4331 @mudit13 @Yaduttam95 @NikSchaefer @Peiprjs @ClaireBookworm @PeterTheSalmon @LordUbuntu @val-hir @Adityabhaskar685 @tiluckdave @VipulChauhan89 @jasAtbiz @samarth641 @ANURAG143d @nesrina-lgr @ZeroQLi @rajanwastaken @NilanchalaPanda @ryanchou-dev @ongzzzzzz @briyandyju09 @sanjus-robotic-studio

**Reply by sampoder** [reply]
@AkulSharma569 @Savalone47 @nikitam25 @Otherwa @Darshan4141 @atul120212 @Tishaa-2002 @ArunodaySingh @avanoxious @YusufHass @mittal-parth @NalinPlad @ZuhairHossain @Abhiharsh-IN @archi-max @JerrySantana @NisargWath @ganning127 @Leo32345 @citricacid2 @karmanyaahm @alexbluo @recursiveforte @SerenityUX @Ostrichization @Shield01 @Rhyzhang @DevIos01 @Athul0491 @allensphilip @shiroyasha9 @phillipe-clinton @julianalg @Parthiv-01 @harsh721827 @hwittenborn @dnyaneshwarrede @Shaikh-Yaqoob @thereavercode @ashutosh7i @AbishekRufusRaj @CheeksTheGeek @VipulChauhan89 @AMSANJEEV28 @ManasG0yal @ANSXX @Nikhil8557 check this out you all!

**Reply by sampoder** [reply]
calling @Jony-Jas @adarssh01 @rimwaka @Darkridersia @DuanBoomer @TirthPatel06 @MHKhan18 @w0110 @Z3r0J @phanturne @Nick67644 @jyotikaisrani @chanderbhanu096 @primitt @Ashwagandhae @briannamagtoto @DEABSTRACT @jayeshrajpoot @sanjay-kv @SamuelATAKE @MohammadAzrak @Devnol @alx-alexpark @bawneet04 @amankumar003 @Abhaykushwah @Redskull-127 @Nakshatraarora @harsha-sam @Kushgupta16 @MitAbhay @minhadona @PasinduDushan @Ritesh-zt0 @BolunThompson @ItsRoy69 @el-tegy @moultamissimane @ravikul505 @Arpan-206 @incchi @InternetRamen @Justine-Njuguna @Sid-WC121 @hariketsheth @SANTHOSH-MAMIDISETTI @sobhanbera @Apoorvgarg-creator @kkg2001 @akashnigam-0! all abroad the outernet bus!

**Reply by sampoder** [reply]
@Younus-Saberi @JosiasAurel @reesericci @satishmane-sam @VS11000 @dem0nsl4yer @Ahmed-aleryani @festusmaithyakcau @poonam-16 @akashnigam-0 @HEYKASHIF @bharath-kumar143 @alexanderhm95 @louremipsum @polypixeldev @FrankOnyemaOrji @anamiikajha @debater-coder @FrankOnyemaOrji @nobiit @Linus-Lu @HenryBass @zhuofan-16 @cfanoulis @tddschn @anishsaraogi @hrithikthakur let's get you all in here as well!

**Comment by hwittenborn** [comment]
I'm so pumped up 💪💪 nerd city is approaching fast

**Reply by sampoder** [reply]
NERD CITY!!!!!!!

**Comment by opticaldrive** [comment]
I think I would probably test out robots.
But first I would have to make the robot. :)

**Comment by SANTHOSH-MAMIDISETTI** [comment]
what to be done now guys!!!!!!!!!!!!!!!!
